spec/services/remove_document_service_spec.rb

```diff
@@ -78,13 +79,40 @@
             explanation: "explanation",
             locale: edition.locale,
             type: "gone",
+            unpublished_at: removal.removed_at,
           },
         )
         described_class.call(edition, removal)
         expect(request).to have_been_requested
       end
     end
 
+    context "when a removal already exists" do
+      it "maintains the existing removed_at time" do
+        existing_removed_at = 2.days.ago
+        existing_removal = build(:removal, removed_at: existing_removed_at)
+        edition = create(:edition, :removed, removal: existing_removal)
+        new_removal = build(:removal,
+                            alternative_url: "/path",
+                            explanatory_note: "explanation",
+                            redirect: true)
+
+        stub_publishing_api_unpublish(
+          edition.content_id,
+          body: {
+            alternative_path: "/path",
+            explanation: "explanation",
+            locale: edition.locale,
+            type: "redirect",
+            unpublished_at: existing_removal.removed_at,
+          },
+        )
```

Comment:
I think for this stub we either want to assert it was requested or not have it defined, as it's already covered by the `stub_any_publishing_api_unpublish` in the before block - I don't really think we need it since we're already checking removed_at is sent in other test.

Reply:
Removed 👍